src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -199,15 +193,15 @@ This section shows how to view and manage a group's roles in the CockroachDB {{
 1. In a group's row, click the three-dots **Action** button, then click **Edit Roles**.
 1. The group's granted roles are shown. Add or remove roles, then click **Confirm**.
 
-      {{site.data.alerts.callout_info}}
-      The fields for inherited roles are read-only, because inherited roles cannot be edited directly. Instead, you must either remove the role from the parent group from which it is inherited, or remove the child group from the parent group.
-      {{site.data.alerts.end}}
+   {{site.data.alerts.callout_info}}
+   The fields for inherited roles are read-only, because inherited roles cannot be edited directly. Instead, you must either remove the role from the parent group from which it is inherited, or remove the child group from the parent group.
+   {{site.data.alerts.end}}
 
 </section>
 
 <section class="filter-content" markdown="1" data-scope="api">
 
-Each pushed group CockroachDB {{ site.data.products.cloud }} is assigned an ID, which is required to manage the group's roles using the CockroachDB {{ site.data.products.cloud }} API or Terraform provider, but is not displayed in the CockroachDB {{ site.data.products.cloud }} Console. A group's ID is returned when you:
+Each group pushed to CockroachDB {{ site.data.products.cloud }} is assigned an ID, which is required to manage the group's roles using the CockroachDB {{ site.data.products.cloud }} API or Terraform provider, but is not displayed in the CockroachDB {{ site.data.products.cloud }} Console. A group's ID is returned when you:
```

Comment:
TODO: Confirm whether a SCIM group ID will now be shown in the UI.

Reply:
From Kyle's comment it sounds like the group ID won't be shown in the UI.